Makefile

```diff
@@ -5,11 +5,11 @@ default: fmt vet errcheck test lint
 # Taken from https://github.com/codecov/example-go#caveat-multiple-files
 .PHONY: test
 test:
-	echo "" > coverage.txt
+	echo "mode: atomic" > coverage.txt
```

Comment:
for my curiosity, why was it required?

Reply:
@varun06 I'm glad you asked :) it was described in the commit message, but happy to repeat it here:

> the existing coverage.txt concatenation included multiple `mode:
  atomic` lines which fail to parse correctly in the `go tool cover`
  tooling, so update Makefile so only one modeline exists

Reply:
🤦‍♂ I just had my tea :)
Thanks for reiterating.